packages/elements/src/containers/API.tsx

```diff
@@ -190,6 +194,20 @@ export const APIImpl: React.FC<APIProps> = props => {
           tryItCorsProxy={tryItCorsProxy}
         />
       )}
+      {layout === 'responsive' && (
```

Comment:
We should make one of those layouts default as `layout` property is optional

Reply:
I'm just wondering if we need a separate option for responsive layout. We might treat responsiveness as enhancement of sidebar layout in this case. This way we could also include it in `StoplightProject`